packages/charts-bar/package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "@ibm-design/charts-bar",
+  "version": "0.1.0",
+  "description": "Sample implementation of the charts bar from the IBM Design Language",
+  "author": "Abby Starnes & Stephane Rodet",
```

Comment:
This can be an array! https://docs.npmjs.com/files/package.json#people-fields-author-contributors